File: Server/Cinema/CinemaApp.Domain/Entities/Seat.cs
namespace CinemaApp.Domain.Entities
{
    public class Seat
    {
        public int Id { get; set; }
        public int HallId { get; set; }
        public Hall Hall { get; set; }
        public int SeatTypeId { get; set; }
        public int SeatGroupId { get; set; }
        public int IndexInsideSeatGroup { get; set; }
        public string RowName { get; set; }
        public short SeatNo { get; set; }
        public SeatType SeatType { get; set; }
        public ICollection<Ticket> Tickets { get; set; }
        public ICollection<TicketSeat> TicketSeats { get; set; }
    }
}

File: Client/cinema-app/src/app/core/models/seat/seat.ts
import { Hall } from '@core/models/hall/hall';
import { SeatType } from '@core/models/seat-type/seat-type';

export interface Seat {
  id: number,
  rowName: string,
  seatNo: number,
  indexInsideSeatGroup: number,
  seatGroupId: number,
  seatTypeId: number,
  hall?: Hall,
  seatType?: SeatType
}
